charts/plugin-br-pix-direct-jd/templates/plugin-br-pix-direct-jd/configmap.yaml

```
@@ -8,6 +8,8 @@ data:
   # -- Default Environment variables for the plugin-br-pix-direct-jd
   NODE_ENV: {{ .Values.pix.configmap.NODE_ENV | default "development" | quote }}
   DEBUG: {{ .Values.pix.configmap.DEBUG | default "false" | quote }}
+  ENVIRONMENT: {{ .Values.pix.configmap.ENVIRONMENT | default "development" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)